pkg/data/management/role_data.go

```diff
@@ -85,6 +85,8 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 	userRole.addNamespacedRule("cattle-global-data").addRule().apiGroups("").resources("secrets").verbs("create")
 
 	rb.addRole("User Base", "user-base").
+		// standard permissions for regular users, on their tokens
```

Comment:
nit:  With this line of code we grant permissions to all tokens. It'd be nice to mention in the comment that we enforce a user can only access their tokens in the Store.

Reply:
Done with commit 73501deeb6e905088f0eab7040de8c26f348d4b9 (rebase!)